packagers/yarn-workspaces/yarn.lock

```diff
@@ -99,12 +99,12 @@ make-error@^1.1.1:
   resolved "https://registry.yarnpkg.com/make-error/-/make-error-1.3.6.tgz#2eb2e37ea9b67c4891f684a1394799af484cf7a2"
   integrity sha512-s8UhlNe7vPKomQhC1qFelMokr/Sc3AgNbso3n74mVPA5LTZwkB9NlXf4XPamLxJE8h0gh73rM94xvwRT2CVInw==
 
-prisma@5.3.0-dev.36:
-  version "5.3.0-dev.36"
-  resolved "https://registry.yarnpkg.com/prisma/-/prisma-5.3.0-dev.36.tgz#84dec32fce7be4bd5c12263f6ad1c277a6ac4bf1"
-  integrity sha512-MEgndZlSJxCeZkkwDwJhikwens3i557e5p2QyGsTkhwVmX62A4lozqO3Wkq09xFKhcI+MGx7brFx7CNEe4+Ijg==
+prisma@5.3.0-dev.38:
+  version "5.3.0-dev.38"
+  resolved "https://registry.yarnpkg.com/prisma/-/prisma-5.3.0-dev.38.tgz#af352fda2f5d487077bd1631dee1fa21b4de889e"
+  integrity sha512-Rmlhqd4+ieqDQyWOSqz+U3s6I0QhwB82enEEFgZJIWIHu+HFk4pPDyxm70kWiNetpQLPB7rHt7jB54i9rWPPQQ==
   dependencies:
-    "@prisma/engines" "5.3.0-dev.36"
```

Comment:
Flakiness with yarn that I just noticed, this is because we run it in parallel to update lockfiles but it does not work very well. Will fix in a separate commit.